pkgs/applications/networking/instant-messengers/qq/default.nix

```diff
@@ -77,7 +78,7 @@ stdenv.mkDerivation {
       --replace "/usr/share" "$out/share"
     makeWrapper $out/opt/QQ/qq $out/bin/qq \
       --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ libGL ]}" \
-      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
+      --add-flags "${customArgs}"
```

Comment:
Maybe it is better to follow `NIXOS_OZONE_WL` variable and provide a customArgs option at the same time.
Reference to [scripts in VSCode](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/editors/vscode/generic.nix#L212).

Reply:
> Maybe it is better to follow `NIXOS_OZONE_WL` variable and provide a customArgs option at the same time. Reference to [scripts in VSCode](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/editors/vscode/generic.nix?rgh-link-date=2024-03-31T09%3A09%3A56Z#L212).

Thanks for suggestion! Merged in latest push.